lib/index.js

```diff
@@ -31,6 +32,36 @@ function getFooterNode() {
   return div;
 }
 
+function getItemValue(item) {
+  return item.place_name
+}
+
+function render(item) {
+  var placeName = item.place_name.split(',');
+  return '<div class="mapboxgl-ctrl-geocoder--suggestion"><div class="mapboxgl-ctrl-geocoder--suggestion-title">' + placeName[0]+ '</div><div class="mapboxgl-ctrl-geocoder--suggestion-address">' + placeName.splice(1, placeName.length).join(',') + '</div></div>';
+}
+
+function getItemValueV6(item) {
+  const { name, place_formatted } = item.properties;
+  return name + (place_formatted ? `, ${place_formatted}` : "");
+}
+
+function renderV6(item) {
+  const { name, place_formatted } = item.properties;
+  return `<div class="mapboxgl-ctrl-geocoder--suggestion">
+      ${
+  name
+    ? `<div class="mapboxgl-ctrl-geocoder--suggestion-title">${name}</div>`
+    : ""
+}
```

Comment:
It looks like the whitespace formatting may be off here.

Reply:
could you specify what do you mean?